### PR TITLE
added blog post about Object.freeze and performance in Vue

### PR DESCRIPTION
**Pull request by hillalex**

### CODE REVIEW

File: content/blog/vue-performance.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+date: 2020-03-01
+title: Handling long arrays performantly in Vuejs
+tags:
+ - Vue
+ - Javascript
+ - performance
+---
+
+
+## TLDR;
```
Comment (richfitz) [inline content/blog/vue-performance.md]: Looks like canonical spelling is `tl;dr` (https://knowyourmeme.com/memes/tldr)

File: content/blog/vue-performance.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+Naomi provides visualisations of HIV epidemic indicators 
+across a country, at a fine-grained regional level. Users can filter the data by many parameters, 
+e.g. age, sex, year, level of of detail (from the whole country down to individual cities.)
+
+<img src="/img/choropeek.gif" alt="Gif of the map widget in Naomi" />
+
+The front-end app is written in [Vue.js](https://vuejs.org/) + [Vuex](https://vuex.vuejs.org/) and for the map plots we 
+are using the excellent [Leaflet](https://leafletjs.com/) library. The HIV indicator data and GeoJSON are returned from
+ an API and are in the  order of 10MB in size; the fitering of data happens in the front-end and the map is redrawn each
```
Comment (richfitz) [inline content/blog/vue-performance.md:32]: "and are _of_ the order of 10..." sounds more natural to my ears but I'm doubting it as I write this...

File: content/blog/vue-performance.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+<img src="/img/memoryheap.png" alt="Screenshot of heap snapshot">
+ 
+## Under the hood
+
+ In Vue, everything is reactive by default. This feels like magic, but what's really going on 
+is that every object (in this case, every item in the array) is having an "observer" property added to it, this is an 
+ instance of the [Dep](https://github.com/vuejs/vue/blob/2.6/src/core/observer/dep.js#L13) class,
+and then having reactive getter and setter functions added for every property: 
+https://github.com/vuejs/vue/blob/2.6/src/core/observer/index.js#L135 *
```
Comment (richfitz) [inline content/blog/vue-performance.md]: For footnotes see `gotchaR-bumps.md` for the `[^1]` syntax we've used elsewhere

**Review by richfitz — COMMENTED** [review]
I think this reads really well and even with no real knowledge of the systems I can follow the motivation and reasoning. Exceptionally minor changes suggested

File: content/blog/vue-performance.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
+---
+author: "Alex Hill"
+date: 2020-03-01
+title: Handling long arrays performantly in Vuejs
```
Comment (EmmaLRussell) [inline content/blog/vue-performance.md]: Usually Vue.js?

**Review by EmmaLRussell — COMMENTED** [review]
Looks good!

**Comment by hillalex** [comment]
Thanks both. I've put the date in the future a bit so we can space posts out (since @richfitz has one to go out also dated 16th, I've put this one to go out on the 23rd)

**Pull request merged**